srcs/client/src/Pages/Game/Game.tsx

```diff
@@ -0,0 +1,115 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+import Canvas from '../../Components/Canvas/Canvas';
+
+const Game = (props: any) => {
+  // Enum
+  enum movement {
+    UP = 0,
+    DOWN,
+  }
+
+  const enum Motive {
```

Comment:
let's just use enums instead of const enums. 
btw I've started to write my enums as:
enum eMovement {...}

I think it's better this way in order to show intent. 

Also just like you, I've stopped using upper case enums like in C

Reply:
Ok yes nice idea, I will include it in the next game PR